builders/tools/precompiles/democracy.md

```diff
@@ -0,0 +1,203 @@
+
+On the next screen, take the following steps:
+
+ 1. Press the triangle icon to reveal the encoded proposal in bytes
+ 2. Copy the encoded proposal - you'll need this when calling the **note_preimage** function in a later step
+
+![Get the encoded proposal](/images/builders/tools/precompiles/democracy/democracy-5.png)
+
+!!! note
+     You should NOT sign and submit the transaction here. You will submit this information via the **note_preimage** function in a later step.  
+
+### Call the Propose Function {: #call-the-propose-function }
+
+1. Expand the Democracy precompile contract to see the available functions
+2. Find the **propose** function and press the button to expand the section
+3. Enter the hash of the proposal
+4. Enter the value in WEI of the tokens to bond. The minimum bond is  {{ networks.moonbase.democracy.min_deposit }} DEV / {{ networks.moonriver.democracy.min_deposit }} MOVR or {{ networks.moonbeam.democracy.min_deposit }} GLMR. For this example 4 DEV or `4000000000000000000` was entered
+5. Press **transact** and confirm the transaction in MetaMask
+
+![Call the propose function](/images/builders/tools/precompiles/democracy/democracy-6.png)
+
+### Submit the Preimage {: #submit-the-preimage }
+
+At this step, you'll take the encoded proposal that you got from [Polkadot.JS Apps](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Fmoonbeam-alpha.api.onfinality.io%2Fpublic-ws#/democracy){target=_blank} and submit it via the **note_preimage** function of the democracy precompile. Despite its name, the preimage is not required to be submitted before the proposal. However, submitting the preimage is required before a proposal can be enacted. To submit the preimage via the **note_preimage** function, take the following steps:
+
+1. Expand the Democracy precompile contract to see the available functions 
+2. Find the **note_preimage** function and press the button to expand the section
+3. Copy the encoded proposal that you noted in the prior section. Note, the encoded proposal is not the same as the preimage hash. Ensure you are are entering the correct value into this field
+4. Press **transact** and confirm the transaction in MetaMask
+
+![Submit the preimage](/images/builders/tools/precompiles/democracy/democracy-7.png)
+
+After your transaction has been confirmed you can return to the Democracy section of [Polkadot.JS Apps](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Fmoonbeam-alpha.api.onfinality.io%2Fpublic-ws#/democracy){target=_blank} to see your proposal listed in the proposal queue.
+
+## Second a Proposal {: #second-a-proposal } 
+
+Seconding a proposal allows it to move to referendum status and requires a bond equivalent to the bond furnished by the proposer. Seconded proposals transition to referendum status once per launch period, which is approximately {{ networks.moonbase.democracy.launch_period.days}} day in Moonbase Alpha and Moonriver, and {{ networks.moonbeam.democracy.launch_period.days}} days in Moonbeam. 
```

Comment:
I would change how the launch periods are listed in case something changes on Moonriver and the values end up being different